test/SparqlBenchmarkRunner-test.ts

```diff
@@ -72,6 +72,116 @@ describe('SparqlBenchmarkRunner', () => {
   describe('run', () => {
     it('runs the whole query set', async() => {
       const results = await runner.run();
+      //
```

Comment:
Do we really need these comments?

Reply:
It was really difficult to modify the test without knowing what the `IResult` where. I had to reconstruct them with the debugger.

Reply:
The problem of this comment is that if the underlying data changes, this comment would also have to be updated.
But we don't have a check in place to make sure that this comment is aligned.
So either we need something to dynamically generate this output, or just remove the comment.

Reply:
I will delete it then, because making this change would be out of the scope of the PR and I guess people can do what I did to get the information.